PepperDashEssentials/Room/Types/EssentialsHuddleVtc1Room.cs

```diff
@@ -298,6 +318,7 @@ void Initialize()
 
 
                 VideoCodec.CallStatusChange += (o, a) => this.InCallFeedback.FireUpdate();
+				VideoCodec.IsReadyChange += (o, a) => this.SetCodecExternalSources(); 

```

Comment:
I think we might want to actually check the args in this event to make sure that the codec is ready.